### DISCUSSION TITLE
Water Hot Plot: Fine-Scale Water Quality Visualization with Plotnine

### DISCUSSION THREAD

**Discussion by UncleCamsWaterPlans** [has2k1/plotnine#830] [2024 Plotnine Contest]
### Authors

Cameron Roberts

### Links

[github](https://github.com/UncleCamsWaterPlans), [linkdin](www.linkedin.com/in/cameron-roberts)

### Full description

Please find my full submission hosted via Github pages [HERE](https://unclecamswaterplans.github.io/water_hot_plot/herbert_catchment_heatmap.html)

### Code repository

**Code repository:** https://github.com/UncleCamsWaterPlans/water_hot_plot

**Comment by AlexWaterboyBezzina** [comment]
This plot is straight igneous🔥

**Comment by has2k1** [comment]
![wc](https://github.com/user-attachments/assets/1725a670-fa87-4d2e-8f67-b85a40946ab0)

**Comment by EconomiCurtis** [comment]
Ahoy @UncleCamsWaterPlans , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Reply by UncleCamsWaterPlans** [reply]
Hello! 
That's great. I'd love that, thank you :)

**Comment by has2k1** [comment]
@UncleCamsWaterPlans, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
